megamek/src/megamek/common/Mech.java

```diff
@@ -3246,6 +3270,34 @@ public TechAdvancement getConstructionTechAdvancement() {
                 .setPrototypeFactions(F_FW).setProductionFactions(F_FW)
                 .setAvailability(RATING_X, RATING_F, RATING_F, RATING_F)
                 .setStaticTechLevel(SimpleTechLevel.ADVANCED), //Superheavy industrial
+            //FIXME: The data is for Superheavy.
```

Comment:
The superheavy tech advancement is more restrictive than the command console, so this can stand as is.